schema/defs-image.json

```diff
@@ -0,0 +1,99 @@
+{
+  "description": "Definitions particular to OpenContainer Image Specification",
+  "definitions": {
+    "mediaType": {
+      "id": "https://opencontainers.org/schema/image/mediaType",
+      "type": "string",
+      "pattern": "^[a-z]+/[0-9a-zA-Z.+]+$"
+    },
+    "digest": {
+      "description": "the cryptographic checksum digest of the object, in the pattern '<hash>:<hexadecimal digest>'",
+      "type": "string",
+      "pattern": "^[a-z0-9]+:[a-fA-F0-9]+$"
+    },
+    "blobObject": {
```

Comment:
where is that defined?

On Fri, Apr 15, 2016 at 1:07 PM Stephen Day <notifications@github.com>
wrote:

> In schema/defs-image.json
> <https://github.com/opencontainers/image-spec/pull/18#discussion_r59933563>
> :
>
> > @@ -0,0 +1,99 @@
> > +{
> > +  "description": "Definitions particular to OpenContainer Image Specification",
> > +  "definitions": {
> > +    "mediaType": {
> > +      "id": "https://opencontainers.org/schema/image/mediaType",
> > +      "type": "string",
> > +      "pattern": "^[a-z]+/[0-9a-zA-Z.+]+$"
> > +    },
> > +    "digest": {
> > +      "description": "the cryptographic checksum digest of the object, in the pattern '<hash>:<hexadecimal digest>'",
> > +      "type": "string",
> > +      "pattern": "^[a-z0-9]+:[a-fA-F0-9]+$"
> > +    },
> > +    "blobObject": {
>
> This is actually called a "Descriptor".
>
> —
> You are receiving this because you modified the open/close state.
>
>
> Reply to this email directly or view it on GitHub
> <https://github.com/opencontainers/image-spec/pull/18/files/64b5003c9985ba8714201f1ecc01460770ab5828#r59933563>
>